web/src/helpers/api.js

```
@@ -82,6 +82,7 @@ export const buildApiPayload = (messages, systemPrompt, inputs, parameterEnabled
 
   const payload = {
     model: inputs.model,
+    group: inputs.group,
     messages: processedMessages,
     group: inputs.group, 
```

Biome (1.9.4)

[error] This property value named group is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named group


(lint/suspicious/noDuplicateObjectKeys)